build-scripts/component-common.gradle

```diff
@@ -50,3 +50,115 @@ dependencies {
     androidTestImplementation libs.test.espresso.core
     androidTestImplementation libs.test.runner
 }
+
+// Shared logic for projects that depend on libmegazord
+//
+// This ensures that libmegazord will be in the library path so that it can be loaded.  It also adds
+// the transitive JNA dependency.
+ext.dependsOnTheMegazord = {
+    dependencies {
+        api project(":full-megazord")
+
+        // Add a JNA dependency, which is required by UniFFI.
+        //
+        // The tricky thing is getting the correct libjnidispatch library. Use the AAR version for
+        // normal builds, since that ensures the various Android libraries get packaged correctly when
+        // we build our own AAR.  Use the JAR version for tests, since that ensures the tests can load
+        // the native version.
+        implementation(libs.jna) {
+            artifact {
+                type = "aar"
+            }
+        }
+        testImplementation(libs.jna) {
+            artifact {
+                type = "jar"
+            }
+        }
```

Comment:
I'm pretty sure the fact this works, means we don't need the `jnaForTest` configuration.  Tell me if I'm missing anything.